### PR TITLE
Open Sourcing Smart City Data

### PR DESCRIPTION
**Pull request by rajones-hntb**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +0,0 @@
-FROM bitwalker/alpine-elixir:1.8.1
```
Comment (ManApart) [inline Dockerfile]: Should this guy really be deleted? I've heard he could be helpful for running/testing locally?

File: README.md [hunk 2]
```diff
@@ -29,7 +33,7 @@ iex> SmartCity.Data.new(%{dataset_id: "a_guid", payload: "the_data", _metadata:
 ```elixir
 def deps do
   [
-    {:smart_city_data, "~> 2.1.4", organization: "smartcolumbus_os"}
+    {:smart_city_data, "~> 2.1.5", organization: "smartcolumbus_os"}
```
Comment (jeffgrunewald) [inline README.md]: we can pull out the org reference once this goes through.

File: Dockerfile (hunk 1 above)
Comment (rajones-hntb) [reply Dockerfile]: its a win / lose situation, it adds more net code that is repeated in the travis file but it does make local testing easier for devs who use the dockerfile that way. We opted to delete it.

File: README.md (hunk 2 above)
Comment (rajones-hntb) [reply README.md]: Thanks, good catch, we changed it.

File: Dockerfile (hunk 1 above)
Comment (ManApart) [reply Dockerfile]: We had kept ours, probably should norm one way or the other. I think I prefer what you guys did.

**Pull request merged**